indra/newview/lldrawpool.cpp

```diff
@@ -651,6 +651,10 @@ void LLRenderPass::pushUntexturedBatch(LLDrawInfo& params)
         return;
     }
 
+    llassert(params.mVertexBuffer);
+    if (!params.mVertexBuffer)
+        return;
+
```

Comment:
This situation doesn appear to be possible.

1. LLDrawInfo gets initialized with the buffer so it always has this buffer from the start and I see no resets or assigments outside of constructor.
2. In each buffer gets used directly before constructing LLDrawInfo, as far as I can tell supplied buffer can't be null otherwise viewer would have crashed before contructing LLDrawInfo.
3. Even if I missed something and null does get passed, based on context it looks like right solution would be to not make LLDrawInfo and just return.
4. Asserting this far is not helpfull, if you need an assert it should be in place where null buffer gets assigned, where we can unwind it to see what's going on.